demo/full/bxf-parser2/src/parser/node_commons/transition.js

```diff
@@ -0,0 +1,23 @@
+import reduceChildren from "../reduce_children";
+
+export default function parseTransition(node) {
+  return reduceChildren(node, (res, name, node) => {
+    switch (name.toLowerCase()) {
+      case "ns1:videotransitions":
+        const videoTransition = reduceChildren(node, (res, name, node) => {
+          switch (name.toLowerCase()) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: "switch" statements should have at least 3 "case" clauses ([javascript:S1301](https://sonarcloud.io/organizations/rx-player/rules#rule_key=javascript%3AS1301))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=366&issues=AWbt7Z39W3Y69bz5aGXM&open=AWbt7Z39W3Y69bz5aGXM)

[ISSUE_KEY]: AWbt7Z39W3Y69bz5aGXM